scripts/qemu-check.sh

```diff
@@ -95,6 +95,14 @@ do
 		SHM_MBOX=qemu-bridge-hp-sram-mem
 		ROM="-r ../sof.git/build_${j}_gcc/src/arch/xtensa/rom-$j.bin"
 	fi
+        if [ $j == "imx8" ]
+        then
+                echo "IMX8"
+                READY_IPC="00 00 00 00 00 00 04 c0"
+                SHM_IPC_REG=qemu-bridge-mu-io
+                #OUTBOX_OFFSET="1000"
+                SHM_MBOX=qemu-bridge-mbox-io
```

Comment:
Do we actually need OUTBOX_OFFSET? If not lets remove it. Leaving code commented can be confusing as the reader doesn't really know what's the intention here.